<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formpurchaseorder</class>
 <widget class="QWidget" name="Formpurchaseorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nopoLabel">
      <property name="text">
       <string>nopo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nopoLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="jangkapengirimanLabel">
      <property name="text">
       <string>jangkapengiriman</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="jangkapengirimanLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="idklienLabel">
      <property name="text">
       <string>idklien</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="idklienLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tglpoLabel">
      <property name="text">
       <string>tglpo</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tglpoLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="statuspoLabel">
      <property name="text">
       <string>statuspo</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="statuspoLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="qytLabel">
      <property name="text">
       <string>qyt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="qytLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="uoMLabel">
      <property name="text">
       <string>UoM</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="uoMLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="idbarangLabel">
      <property name="text">
       <string>idbarang</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="idbarangLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="idgudangLabel">
      <property name="text">
       <string>idgudang</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="idgudangLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="hargaLabel">
      <property name="text">
       <string>harga</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="hargaLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
